docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -125,7 +70,7 @@ Each element of the `pages` array has the following structure:
 | **`isLast`**  | boolean  | If this page is the last page. Useful to add separators between pages, but avoid adding it for the last one. |
 | **`Wrapper`** | React.FC | The Wrapper component that should wrap the real `Archive` component.                                         |
 
```

Comment:
In the `object properties` table above, change:
"If the next page returned an error. Useful to try again. "
to:
"If the next page returned an error. Useful to provide functionality (e.g. a button) to enable the user to try again. "